ipatests/test_integration/test_acme.py

```diff
@@ -53,17 +61,22 @@ class TestACME(IntegrationTest):
     num_replicas = 0
     num_clients = 1
 
+    @classmethod
+    def install_acme_client(cls):
+        # install acme client packages
+        if not skip_certbot_tests:
+            tasks.install_packages(cls.clients[0], ["certbot"])
+        if not skip_mod_md_tests:
+            tasks.install_packages(cls.clients[0], ["mod_md"])
+
     @classmethod
     def install(cls, mh):
         # cache the acme service uri
         acme_host = f'{IPA_CA_RECORD}.{cls.master.domain.name}'
         cls.acme_server = f'https://{acme_host}/acme/directory'
```

Comment:
This common behaviour can also be extracted to `install_acme_client` classmethod.